binaryop/funcs.go

```diff
@@ -107,3 +107,19 @@ func Ifnot(left, right float64) float64 {
 	}
 	return nan
 }
+
+// And returns 1 if left and right is more than 0. Otherwise 0 is returned.
+func And(left, right float64) float64 {
+	if left > 0 && right > 0 {
```

Comment:
The sign of the numbers should not affect the and.

Reply:
Also, for an and without the modifier `bool`, the returned value is left.

Reply:
> Also, for an and without the modifier `bool`, the returned value is left.

then 0 and 1 should return 0，it's a little bit weird

Reply:
> The sign of the numbers should not affect the and.

now usigned number return 1, thanks bro.